hieradata/production.yaml

```
@@ -216,6 +217,11 @@ govuk_crawler::ssh_keys:
     key: 'AAAAB3NzaC1yc2EAAAADAQABAAABAQDHi69quH4i1jY4uVLuIYOoUi7yc+6/fJjDKJciWIIgyH1rCSGtTXrfcJQHQKxKEyndQ+LoUs8krImD9CtzQTvoqdXYKu+XgpjqiZ5HfQhx55ZoMCXnx9vqCwGtx2LwH4PaBZRvsNJbbY+sG3W13eXkgwQnIcI2FgGpDDJQ9lfxSNzt6jMJQuvAhhDrKtZEsA56AznPIRd1mIHmuZ6gILBzJmDw5EIpqQhSb4+NrNTNnGxHyFkZISTMG0mx9lRtlyS1SMZVoPJQbShSEScKMXRlacrpkiNOTvrGDrhsLjTUtT5g1bo4XMFRubReAwFCeTZI0Qv1Bvpzj8V9c8B4+p6b'
     type: 'ssh-rsa'
 
+govuk::apps::govuk_crawler_worker::root_urls:
+  - 'https://assets.digital.cabinet-office.gov.uk/'
+  - 'https://assets.publishing.service.gov.uk/'
```

How about using the app domain here to remove duplication?

```suggestion
  - "https://assets.%{hiera('app_domain')}"
```

---

Could do, but I think I'd rather emphasise it's hard-coded. Also, we can't do it for all of them consistently :-|.

---

`%{hiera('app_domain')}` is referenced consistency throughout `production.yaml`. If we were to change the configure app domain we would want that change to apply here too.

---

Done.